test/parallel/test-child-process-recv-handle.js

```diff
@@ -15,49 +15,49 @@ else
 function master() {
   // spawn() can only create one IPC channel so we use stdin/stdout as an
   // ad-hoc command channel.
-  var proc = spawn(process.execPath, [__filename, 'worker'], {
+  const proc = spawn(process.execPath, [__filename, 'worker'], {
     stdio: ['pipe', 'pipe', 'pipe', 'ipc']
   });
-  var handle = null;
-  proc.on('exit', function() {
+  let handle = null;
+  proc.on('exit', () => {
     handle.close();
   });
-  proc.stdout.on('data', function(data) {
-    assert.equal(data, 'ok\r\n');
+  proc.stdout.on('data', common.mustCall((data) => {
+    assert.strictEqual(data.toString(), 'ok\r\n');
     net.createServer(common.fail).listen(0, function() {
       handle = this._handle;
       proc.send('one');
       proc.send('two', handle);
       proc.send('three');
       proc.stdin.write('ok\r\n');
     });
-  });
+  }));
   proc.stderr.pipe(process.stderr);
 }
 
 function worker() {
   process._channel.readStop();  // Make messages batch up.
   process.stdout.ref();
   process.stdout.write('ok\r\n');
-  process.stdin.once('data', function(data) {
-    assert.equal(data, 'ok\r\n');
+  process.stdin.once('data', common.mustCall((data) => {
+    assert.strictEqual(data.toString(), 'ok\r\n');
     process._channel.readStart();
-  });
-  var n = 0;
-  process.on('message', function(msg, handle) {
+  }));
+  let n = 0;
+  process.on('message', common.mustCall((msg, handle) => {
     n += 1;
     if (n === 1) {
-      assert.equal(msg, 'one');
-      assert.equal(handle, undefined);
+      assert.strictEqual(msg, 'one');
+      assert.strictEqual(handle, undefined);
     } else if (n === 2) {
-      assert.equal(msg, 'two');
-      assert.equal(typeof handle, 'object');  // Also matches null, therefore...
-      assert.ok(handle);                      // also check that it's truthy.
+      assert.strictEqual(msg, 'two');
+      assert.strictEqual(typeof handle, 'object'); // Also matches null,
```

Comment:
Maybe to simplify we could combine this and the following assertion like `assert.ok(handle !== null && typeof handle === 'object');`